### PR TITLE
feat: restructure installation and add install steps for PVE and LXC

### PR DESCRIPTION
**Pull request by cal-weng**
- [x] add guides for PVE and LXC on PVE
- [x] Merge or Linux based systems  
- [x] edit related config files

### CODE REVIEW

**Review by fnalways — CHANGES_REQUESTED** [review]
The on-page toc does not update when switching sub-tabs. Do not merge this PR until it's fixed.

File: docs/manual/get-started/install-olares-linux.md [hunk 1]
```diff
@@ -28,6 +41,239 @@ bash olares-uninstall.sh
...
+### Configure the LXC environment
+
+1. Create the LXC container using the following script:
+
+   ::: tip Note
+   To create a container, you need to assign it a unique container ID. In this guide, we use `16553`, but you can replace it with any available numeric ID. Make sure to update all commands and configurations accordingly.
+   :::
+
+   ::: tip
+   If you want to install Olares on an existing LXC container. Skip to step 2 directly. Make sure you use the corresponding container ID.
```
Comment (cal-weng) [inline docs/manual/get-started/install-olares-linux.md:114]:
```suggestion
To install Olares on an existing LXC container, skip to step 2 directly. Make sure you use the corresponding container ID.
```

File: docs/manual/get-started/install-olares-general-linux.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+After uninstalling, retry the installation by running the original installation command.
+:::
+
+## Prepare Wizard URL
+
+At the end of the installation process, you will need to provide some information:
+
+1. Enter the root user password.
+
+   ![Enter password](/images/manual/get-started/enter-root-user-password.png)
```
Comment (fnalways) [inline docs/manual/get-started/install-olares-general-linux.md:38]: In chinese version, we use note for this step. Should we align here (and also for other linux-based system)?

File: docs/zh/manual/get-started/install-olares-lxc.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+      # 创建缺失的目录
+      mkdir -p /lib/modules
+
+      # 更新 PATH 环境变量
+      echo 'export PATH="/usr/local/bin:$PATH"' >> /root/.bashrc
+      source ~/.bashrc
+      
+      # 退出 LXC
+      exit
+      ```
```
Comment (fnalways) [inline docs/zh/manual/get-started/install-olares-lxc.md:113]: Adjust indentation

**Review by fnalways — COMMENTED** [review]
Comments also apply to chinese docs.

File: docs/zh/manual/get-started/install-olares-lxc.md [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+
+### 配置 LXC 环境
+
+1. 使用以下命令创建 LXC 容器：
+
+   ::: tip 注意
+   要创建容器，必须分配一个唯一的 **容器 ID**。在本指南中，我们使用 `16553`，但你可以将其替换为任何可用的数字 ID，并在所有相关命令和配置中更新此 ID。
+   :::
+
+   ::: tip 提示
```
Comment (cal-weng) [inline docs/zh/manual/get-started/install-olares-lxc.md:33]:
```suggestion
   ::: tip 安装至已有 LXC 容器
```

File: docs/zh/manual/get-started/install-olares-lxc.md (hunk 3 above)
Comment (cal-weng) [inline docs/zh/manual/get-started/install-olares-lxc.md:113]:
```suggestion
   ```bash 
   # 启动容器
   pct start 16553

   # 进入容器
   pct enter 16553

   # 创建缺失的目录
   mkdir -p /lib/modules

   # 更新 PATH 环境变量
   echo 'export PATH="/usr/local/bin:$PATH"' >> /root/.bashrc
   source ~/.bashrc
      
   # 退出 LXC
   exit
   ```
```

File: docs/zh/manual/get-started/install-olares-lxc.md [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+   ```bash
+   mkdir -p /root/.olares/images /root/.olares/pkg
+   ```
+-  `debian-12-standard_12.7-1_amd64.tar.zst`的容器模板（CT），可从 PVE [镜像仓库](http://download.proxmox.com/images/system/)下载。
+
+### 配置 LXC 环境
+
+1. 使用以下命令创建 LXC 容器：
+
+   ::: tip 注意
```
Comment (cal-weng) [inline docs/zh/manual/get-started/install-olares-lxc.md:29]:
```suggestion
   ::: tip 指定唯一容器 ID
```

File: docs/zh/manual/get-started/install-olares-lxc.md [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
+请确保设备满足以下配置要求：
+
+- CPU：4 核及以上
+- 内存：8GB 及以上（可用内存）
+- 存储：64GB 及以上（可用磁盘空间）
+- 支持的系统版本：
+  - PVE: 8.2.2
+  - LXC 系统：Debian 12
+
+::: tip 注意
```
Comment (cal-weng) [inline docs/zh/manual/get-started/install-olares-lxc.md:12]:
```suggestion
::: tip 版本兼容性
```

File: docs/zh/manual/get-started/install-olares-pve.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
+## <span class="h2-border-none">系统要求</span>
+
+请确保设备满足以下配置要求：
+
+- CPU：4 核及以上
+- 内存：8GB 及以上（可用内存）
+- 存储：64GB 及以上（可用磁盘空间）
+- 支持的系统版本：PVE 8.2.2
+
+::: tip 注意
```
Comment (cal-weng) [inline docs/zh/manual/get-started/install-olares-pve.md:10]:
```suggestion
::: tip 版本兼容性
```

File: docs/zh/manual/get-started/install-olares-general-linux.md [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+请确保设备满足以下配置要求：
+
+- CPU：4 核及以上
+- 内存：8GB 及以上（可用内存）
+- 存储：64GB 及以上（可用磁盘空间）
+- 支持的系统版本：
+    - Ubuntu 20.04 LTS 及以上
+    - Debian 11 及以上
+
+::: tip 注意
```
Comment (cal-weng) [inline docs/zh/manual/get-started/install-olares-general-linux.md:12]:
```suggestion
::: tip 版本兼容性
```

File: docs/zh/manual/get-started/install-olares-lxc.md [hunk 9]
```diff
@@ -0,0 +1,202 @@
+## <span class="h2-border-none">系统要求</span>
```
Comment (cal-weng) [inline docs/zh/manual/get-started/install-olares-lxc.md:1]:
```suggestion
## <span class="h2-border-none">系统要求</span>

LXC（Linux 容器）是一种轻量级的虚拟化技术，可以在隔离的容器中运行应用程序。在 PVE 环境下使用 LXC 部署 Olares 能够避免完整虚拟机的额外开销，提供了一种更高效的部署方式。
```

File: docs/manual/get-started/install-olares-lxc.md [hunk 10]
```diff
@@ -0,0 +1,219 @@
...
+LXC (Linux Containers) is a lightweight virtualization method that runs applications in isolated containers. When used on PVE, it enables an efficient way to deploy Olares without the overhead of a full virtual machine.
+
+Make sure your device meets the following requirements.
+
+- CPU: 4 cores or above
+- RAM: 8GB or above (available memory)
+- Storage: 64GB or above (available disk space)
+- Supported systems:
+    - PVE 8.2.2
+    - Linus container: Debian 12 (for existing LXC containers on PVE)
```
Comment (cal-weng) [inline docs/manual/get-started/install-olares-lxc.md:12]:
```suggestion
    - Linux container: Debian 12 (for existing LXC containers on PVE)
```

File: docs/zh/manual/get-started/install-olares-lxc.md [hunk 11]
```diff
@@ -0,0 +1,203 @@
...
+LXC（Linux 容器）是一种轻量级的虚拟化技术，可以在隔离的容器中运行应用程序。在 PVE 环境下使用 LXC 部署 Olares 能够避免完整虚拟机的额外开销，提供了一种更高效的部署方式。
+
+请确保设备满足以下配置要求：
+
+- CPU：4 核及以上
+- 内存：8GB 及以上（可用内存）
+- 存储：64GB 及以上（可用磁盘空间）
+- 支持的系统版本：
+  - PVE: 8.2.2
+  - LXC 系统：Debian 12
```
Comment (cal-weng) [inline docs/zh/manual/get-started/install-olares-lxc.md:12]:
```suggestion
  - LXC 容器系统：Debian 12
```

**Review by fnalways — APPROVED** [review]

**Pull request merged**